frontend/src/employee-frontend/components/Reports.tsx

```diff
@@ -194,7 +194,13 @@ function Reports() {
             </ReportItem>
           </RequireRole>
           <RequireRole
-            oneOf={['ADMIN', 'SERVICE_WORKER', 'DIRECTOR', 'REPORT_VIEWER']}
+            oneOf={[
+              'ADMIN',
+              'SERVICE_WORKER',
+              'DIRECTOR',
+              'REPORT_VIEWER',
+              'UNIT_SUPERVISOR'
+            ]}
```

Comment:
Would it be reasonable to add `getPermittedActions` for the reports? 🤔

Reply:
It would probably have to be rewritten completely when access control dsl is finished. So I don't think it's worth it, but I'm not completely against it.

Reply:
Yeah, I'll do that.